vm/src/stdlib/os.rs

```diff
@@ -2422,6 +2461,12 @@ fn extend_module_platform_specific(vm: &VirtualMachine, module: &PyObjectRef) {
         "_COPYFILE_DATA" => ctx.new_int(COPYFILE_DATA),
         "_fcopyfile" => ctx.new_function(os_fcopyfile),
     });
+
+    #[cfg(all(windows, target_env = "msvc"))]
+    extend_module!(vm, module, {
+        "wait" => ctx.new_function(os_wait),
+        "waitpid" => ctx.new_function(os_waitpid),
+    })
```

Comment:
This is placed in unix `extend_module_platform_specific` for now. Moving this code a few lines below will be perfect.